src/control/server/instance_drpc.go

```diff
@@ -109,7 +109,7 @@ func drespToMemberResult(rank system.Rank, dresp *drpc.Response, err error, tSta
 	resp := &mgmtpb.DaosResp{}
 	if err = proto.Unmarshal(dresp.Body, resp); err != nil {
 		return system.NewMemberResult(rank,
-			errors.WithMessagef(err, "rank %s dRPC unmarshal failed", &rank),
+			errors.Errorf("rank %s dRPC unmarshal failed", &rank),
```

Comment:
Ah, good catch. I was having some trouble with the unit tests because the new protobuf releases intentionally vary the error string in order to dissuade string comparisons. I'd meant to restore this and fix the tests. I'll do it in a follow-on (#3863).